modules/rosa-autorepair-cli.adoc

```diff
@@ -0,0 +1,80 @@
+// Module included in the following assemblies:
+//
+// * rosa_cluster_admin/rosa_nodes/rosa-managing-worker-nodes.adoc
+// * nodes/rosa-managing-worker-nodes.adoc
+
+
+:_mod-docs-content-type: PROCEDURE
+[id="rosa-autorepair-cli_{context}"]
+= Configuring machine pool AutoRepair using the ROSA CLI
+
+You can configure machine pool AutoRepair for your {product-title} cluster by using the ROSA CLI.
+
+
+.Prerequisites
+
+
+* You installed and configured the latest AWS (`aws`), ROSA (`rosa`) on your workstation.
```

Comment:
Do you need to install the AWS CLI for this procedure? It seems only ROSA CLI is used. The ROSA docs seem a bit inconsistent about this.
This seems to be the most used format when only requiring `rosa`:
```suggestion
* You installed and configured the latest {product-title} (ROSA) CLI, `rosa`, on your workstation.
```
At the very least:
```suggestion
* You installed and configured the latest AWS (`aws`) and ROSA (`rosa`) CLIs on your workstation.
```